<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parse from TXT (extract from PDF) to TXT</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="lWPalabrasTabu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>321</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEPalabraTabu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtAnyadirTabu">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add phrase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtEliminarTabu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove phrase</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEArchivoOrigen">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbFrasesTabu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Phrases to delete in up or down margins:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbArchivoOrigen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Text file to edit:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtArchivoOrigen">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtArchivoGuardar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEArchivoGuardar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbArchivoGuardar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Save in:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtTransformar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply edit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>356</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lEArchivoOrigen</tabstop>
  <tabstop>pBtArchivoOrigen</tabstop>
  <tabstop>lEPalabraTabu</tabstop>
  <tabstop>pBtAnyadirTabu</tabstop>
  <tabstop>lWPalabrasTabu</tabstop>
  <tabstop>pBtEliminarTabu</tabstop>
  <tabstop>lEArchivoGuardar</tabstop>
  <tabstop>pBtArchivoGuardar</tabstop>
  <tabstop>pBtTransformar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
